x-pack/plugins/ml/common/util/job_utils.ts

```diff
@@ -43,9 +49,13 @@ export function calculateDatafeedFrequencyDefaultSeconds(bucketSpanSeconds: numb
   return freq;
 }
 
+export function isTimeSeriesViewJob(job: CombinedJob): boolean {
+  return getSingleMetricViewerJobErrorMessage(job) === undefined;
+}
+
 // Returns a flag to indicate whether the job is suitable for viewing
 // in the Time Series dashboard.
-export function isTimeSeriesViewJob(job: CombinedJob): boolean {
+export function isTimeSeriesViewableJob(job: CombinedJob): boolean {
```

Comment:
I noticed that some of my jobs are greyed out in the job selector in the Single Metric Viewer when they shouldn't be. Is this because some of the logic in here is not quite correct? Jobs which don't use partitioning fields are being disabled here:

![image](https://user-images.githubusercontent.com/7405507/100772039-047a6180-33f7-11eb-872f-526720b0b6e3.png)

Reply:
Removed here https://github.com/elastic/kibana/pull/84594/commits/15cc5ed4c8a053403914709ee35a321241d99294